packages/editor/src/theme.js

```diff
@@ -0,0 +1,39 @@
+export default {
+  initialColorMode: 'light',
+  colors: {
+    text: '#000',
+    background: '#fff',
+    primary: '#33e',
+    secondary: '#119',
+    muted: '#e6e6e6',
+    highlight: '#ffffcc',
+    gray: '#777',
+    purple: '#609',
+    modes: {
+      dark: {
+        text: '#fff',
+        background: '#060606',
+        primary: '#e0f ',
+        secondary: '#3cf',
+        muted: '#252525',
+        highlight: '#ffffcc',
+        gray: '#999',
+        purple: '#c0f',
```

Comment:
I haven't looked into it at all, but do these colors come from official devtools? Would be nice to handle styles however they have baked in for Chrome

Reply:
These just values just come from the website (with minor tweaks). From what I've seen, you have to make your own styles as chrome just applies the same default styles as it does to a website.